enterprise/server/util/vsock/vsock.go

```diff
@@ -0,0 +1,134 @@
+package vsock
+
+import (
+	"bufio"
+	"context"
+	"fmt"
+	"math"
+	"net"
+	"os"
+	"strings"
+	"syscall"
+	"unsafe"
+
+	"github.com/buildbuddy-io/buildbuddy/server/util/status"
+	"golang.org/x/sys/unix"
+
+	libVsock "github.com/mdlayher/vsock"
+)
+
+const (
+	// Corresponds to VHOST_VSOCK_SET_GUEST_CID in vhost.h
+	ioctlVsockSetGuestCID = uintptr(0x4008AF60)
+
+	// 0, 1 and 2 are reserved CIDs, see http://man7.org/linux/man-pages/man7/vsock.7.html
+	minCID = 3
+	maxCID = math.MaxUint32
+
+	DefaultPort = 25415
```

Comment:
Maybe `DefaultHostPort`? (maybe also document where the number come from -- is it firecracker-specific or is it standardized?)

Reply:
It's just a made-up, high-numbered, unused port, our stuff will use.

Reply:
Also was once my zipcode.